src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -504,6 +505,50 @@ public bool IsIronPythonDialogDisabled
             }
         }
 
+        //private ICSharpCode.AvalonEdit.TextEditorOptions editTextOptions = new ICSharpCode.AvalonEdit.TextEditorOptions() {
+        //    ConvertTabsToSpaces = false
+        //};
+        private DynamoTextOptions editTextOptions = new DynamoTextOptions();
+
+        /// <summary>
+        /// Gets the text editor options for python script editor.
+        /// </summary>
+        public DynamoTextOptions TextOptions
```

Comment:
maybe this property name should more clearly describe that it's about python? Especially if it's public - what about PythonScriptEditorTextOptions?